src/CTA.WebForms2Blazor/ControlConverters/LabelControlConverter.cs

```diff
@@ -9,19 +11,28 @@ public class LabelControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["id"] = "for"
+                };
```

Comment:
`id` and `for` should be direct translations so this map can probably be empty for now. This update might not give you an exact replica of eShopOnBlazor, but that's ok